lib/Test/Class/Moose/Role/Executor.pm

```diff
@@ -337,6 +337,7 @@ sub test_classes {
 
     if ( my $classes = $self->test_configuration->test_classes ) {
         if (@$classes) {    # ignore it if the array is empty
+            return shuffle(@$classes) if $self->test_configuration->randomize_classes;
```

Comment:
We don't sort here so I'm not sure we should shuffle either. I think if people explicitly specify classes we should just use the order they gave us.